### ISSUE TITLE
Falcon assemble genome size double then estimated

### ISSUE THREAD

**Issue by lijingjing1** [PacificBiosciences/FALCON#523]
Hi all
     We have a genome size about 600 Mb with high heterozygosity about 3%,and we have 42 Gb,N50 11 Kb pacbio data.
    I got this genome size 1.36 Gb,N50 60 Kb a very bad result with falcon,but we use another assemble software wtdbg (https://github.com/ruanjue/wtdbg) to assemble with those pacbio data it give a result 545 Mb,N50 1.1 Mb.         
   I want konw what make those issue to happen assemble large then estimated and N50 much smaller?Can we get a more better result with falcon if we continue to sequence pacbio data.

**Comment by pb-jchin** [comment]
I am not familiar the algorithm behind wtdbg. I can't comment on that. Without understand the nature of such such genome by looking at the data in detail, it will be hard to guess what is going.

**Comment by yangjie4546** [comment]
yeah, I got the same trouble how can I select the parameters to assemble the high heterozygosity genome? When I change my parameters and got larger N50, I got the larger genome size at the same time, maybe because the heterozygosity region. Similarly, the smaller genome, the smaller N50. So please give some advices for the problem. Thanks!
